### DISCUSSION TITLE
Kms can get expensive

### DISCUSSION THREAD

**Discussion by hopkinsnji** [aws-solutions/quota-monitor-for-aws#185] [General]
Hey all, I'm running this in over a hundred accounts across a couple of regions, KMS costs do add up. Does anyone know of a way to avoid using CMKs?

**Comment by abewub** [comment]
We have added a new template parameter (named 'Report OK Notifications') to suppress the reporting of OK messages in [v6.2.5](https://github.com/aws-solutions/quota-monitor-for-aws/releases/tag/v6.2.5). That should reduce the KMS usage fees by a lot.
